contracts/Bulker.sol

```diff
@@ -5,10 +5,17 @@ import "./CometInterface.sol";
 import "./ERC20.sol";
 import "./IWETH9.sol";
 
+interface IClaimable {
```

Comment:
does the Rewards contract need to explicitly implement this interface now?

Reply:
Nope, not explicitly, assuming you mean extending `IClaimable` directly in the Rewards contract. The only expectation is that the Rewards contract implements the functions defined in `IClaimable`, otherwise the call from Bulker will revert.